### PR TITLE
Add SleepECG

### PR DESCRIPTION
**Pull request by hoechenberger**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hoechenberger** [comment]
@conda-forge/staged-recipes, this PR is ready for review

`LICENSE` is currently vendored, but will be included in the next upstream release: https://github.com/cbrnr/sleepecg/pull/17

File: recipes/sleepecg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  commands:
+    - pip check
+
+about:
+  home: https://sleepecg.readthedocs.io/
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'A toolbox for sleep stage classification using ECG data'
+  dev_url:   https://github.com/cbrnr/sleepecg
```
Comment (hoechenberger) [inline recipes/sleepecg/meta.yaml:48]:
```suggestion
  dev_url: https://github.com/cbrnr/sleepecg
```

File: recipes/sleepecg/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy >=1.20.0
```
Comment (xhochy) [inline recipes/sleepecg/meta.yaml:23]: Stick with the oldest possible one here.
```suggestion
    - numpy 1.20.*
```

**Comment by hoechenberger** [comment]
Thanks @xhochy, I've applied your suggested change.

**Pull request merged**